clients/python/llmengine/data_types.py

```diff
@@ -255,18 +255,16 @@ class CompletionStreamV1Response(BaseModel):
 
 
 class CreateFineTuneJobRequest(BaseModel):
+    model: str
```

Comment:
Curious, did you make these changes by hand, or did you copy from the server? I think @phil-scale tried to set up a script to do that but it's not in place yet.

In any case, we should manually copy from the server for now.

Also, does this PR attempt to solve https://app.shortcut.com/scaleai/story/782421/finetunejob-should-be-renamed-to-finetune-in-documentation?